README.md

```diff
@@ -196,17 +196,24 @@ repeating the mistakes of the past.
       if (isRunningInApp)
       ```
 
-#### Write [Minimum Viable Documentation](https://mobify.atlassian.net/wiki/pages/viewpage.action?pageId=50266690#OurSoftwareDesign&DevelopmentProcess-MinimumViableDocumentation)
-- All modules/libraries/plugins/etc. should have documentation that explains:
-    - Setup
-    - Usage
-    - Deployment
-    - Testing
-    - Contributing
-    - Roadmap/Changelog
-- Good documentation enables others to get up and running without your involvement.
-- Documentation should be updated as you write your software (or [even before you write it](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)))
-- You should be able to answer the majority of questions from users with a link to a document.
+#### Write a README
```

Comment:
Changed to "Write Documentation" because at the end of the day... that's really what we want : D Maybe "Write *Great* Documentation" is more aspirational?